bindings/wasm/lib/bindings.ts

```diff
@@ -20,6 +20,15 @@ const callUtilsMethod = (method: __UtilsMethods__): any => {
     }
 };
 
+// Allow bigint to be serialized as hex string.
+//
+// Note:
+// Serializing `bigint` to a different format, e.g. to decimal number string
+// must be done manually.
```

Comment:
Are there not currently places where this would then have to be changed?

Reply:
Or maybe another question, this isn't inherited from the nodejs bindings ?

Reply:
Yes, only `bindings.ts` is different between Wasm and Node.js. In other places it's inherited from Node.js

Reply:
So if this was somewhere else than in `bindings.ts`, we wouldn't have to duplicate it ? Shouldn't we just move it then?

Reply:
I moved it to `index.ts`, probably `bindings.ts` fits better, but in `index.ts` we don't need to duplicate.